tests/copy.bats

```diff
@@ -668,3 +668,97 @@ parents/y/b.txt"
   run_buildah run $ctr stat -c %u:%g /random-file-6
   assert 789:789
 }
+
+@test "copy-link-flag" {
+  createrandom ${TEST_SCRATCH_DIR}/randomfile
+  createrandom ${TEST_SCRATCH_DIR}/other-randomfile
+
+  run_buildah from $WITH_POLICY_JSON scratch
+  cid=$output
+  run_buildah mount $cid
+  root=$output
+  
+  run_buildah config --workingdir=/ $cid
+  run_buildah copy --link $cid ${TEST_SCRATCH_DIR}/randomfile
+  
+  mkdir $root/subdir
+  run_buildah copy --link $cid ${TEST_SCRATCH_DIR}/randomfile ${TEST_SCRATCH_DIR}/other-randomfile /subdir
+  
+  run_buildah unmount $cid
+  run_buildah commit $WITH_POLICY_JSON $cid containers-storage:copy-link-image
+
+  run_buildah inspect --type=image copy-link-image
+  layers=$(echo "$output" | jq -r '.OCIv1.rootfs.diff_ids | length')
+  assert "$layers" -ge 2 "Expected at least 2 layers from 2 --link operations, but found $layers"
+
+  run_buildah from $WITH_POLICY_JSON copy-link-image
+  newcid=$output
+  run_buildah mount $newcid
+  newroot=$output
+  
+  test -s $newroot/randomfile
+  cmp ${TEST_SCRATCH_DIR}/randomfile $newroot/randomfile
+  test -s $newroot/subdir/randomfile
+  cmp ${TEST_SCRATCH_DIR}/randomfile $newroot/subdir/randomfile
+  test -s $newroot/subdir/other-randomfile
+  cmp ${TEST_SCRATCH_DIR}/other-randomfile $newroot/subdir/other-randomfile
+}
+
+@test "copy-link-directory" {
+  mkdir -p ${TEST_SCRATCH_DIR}/sourcedir
+  createrandom ${TEST_SCRATCH_DIR}/sourcedir/file1
+  createrandom ${TEST_SCRATCH_DIR}/sourcedir/file2
+
+  run_buildah from $WITH_POLICY_JSON scratch
+  cid=$output
+  
+  run_buildah config --workingdir=/ $cid
+  run_buildah copy --link $cid ${TEST_SCRATCH_DIR}/sourcedir /destdir
+  
+  run_buildah commit $WITH_POLICY_JSON $cid containers-storage:copy-link-dir-image
```

Comment:
```suggestion
  run_buildah commit $WITH_POLICY_JSON $cid copy-link-dir-image
```
The `containers-storage:` is implicit, and we don't tend to specify it in tests.